src/GraphMap/Map.cs

```diff
@@ -0,0 +1,289 @@
+using System.Drawing;
+using System.Numerics;
+using System.Text;
+using System.Xml.Schema;
+
+public class Map
+{
+    private const ConsoleColor EMPTY_COLOR = ConsoleColor.Black;
+    private const string EMPTY_SYMBOL = " \u25FC";
+    private const ConsoleColor BORDER_COLOR = ConsoleColor.White;
+    private const string BORDER_SYMBOL = " \u25AA";
+    private const string LOCATION_SYMBOL = " \u25C8";
+    private const ConsoleColor WAY_COLOR = ConsoleColor.DarkYellow;
+    private const string WAY_SYMBOL = " \u25AA";
+    private const string DIRECTION_SYMBOL_UP = " \u2B61";
+    private const string DIRECTION_SYMBOL_UPRIGHT = " \u2B67";
+    private const string DIRECTION_SYMBOL_RIGHT = " \u2B62";
+    private const string DIRECTION_SYMBOL_DOWNRIGHT = " \u2B68";
+    private const string DIRECTION_SYMBOL_DOWN = " \u2B63";
+    private const string DIRECTION_SYMBOL_DOWNLEFT = " \u2B69";
+    private const string DIRECTION_SYMBOL_LEFT = " \u2B60";
+    private const string DIRECTION_SYMBOL_UPLEFT = " \u2B66";
+
+
+    List<Node> nodes;
+    List<Edge> edges;
+
+    public Node currentNode;
+
+    private Vector2 mapSize;
+
+    /// <summary>
+    /// Contains the symbol and color at a certain position.
+    /// </summary>
+    private Tuple<string, ConsoleColor>[,] mapSymbols;

```

Comment:
Sort the variables of the class according to visibility. Maybe add the `private` modifier to nodes and edges.

Reply:
Added and done